src/test/java/uk/gov/pay/api/json/RequestJsonParserTest.java

```diff
@@ -37,40 +37,42 @@ public void parsePaymentRequest_withReturnUrl_shouldParseSuccessfully() throws E
         CreatePaymentRequest createPaymentRequest = parsePaymentRequest(jsonNode);
 
         assertThat(createPaymentRequest, is(notNullValue()));
-        assertThat(createPaymentRequest.getAmount(), is(-1000));
+        assertThat(createPaymentRequest.getAmount(), is(1000));
         assertThat(createPaymentRequest.getReference(), is("Some reference"));
         assertThat(createPaymentRequest.getDescription(), is("Some description"));
         assertThat(createPaymentRequest.getReturnUrl(), is("https://somewhere.gov.uk/rainbow/1"));
     }
 
     @Test
-    public void parsePaymentRequest_withReturnUrlAndLanguage_shouldParseSuccessfully() throws Exception {
+    public void parsePaymentRequest_withReturnUrlAndLanguageAndDelayedCapture_shouldParseSuccessfully() throws Exception {
         // language=JSON
         String payload = "{\n" +
-                "  \"amount\": -1000,\n" +
+                "  \"amount\": 1000,\n" +
                 "  \"reference\": \"Some reference\",\n" +
                 "  \"description\": \"Some description\",\n" +
                 "  \"return_url\": \"https://somewhere.gov.uk/rainbow/1\",\n" +
-                "  \"language\": \"en\"\n" +
+                "  \"language\": \"en\",\n" +
+                "  \"delayed_capture\": false\n" +
```

Comment:
I think I’d prefer if this were `true` just to make us more confident that we’re not somehow relying on the default value of a `boolean` field being `false`